src/test/java/teammates/test/cases/webapi/GetStudentActionTest.java

```diff
@@ -20,16 +29,178 @@ protected String getRequestMethod() {
         return GET;
     }
 
+    private void assertStudentDataMatches(StudentData studentData, StudentAttributes student, boolean isInstructor) {
+        assertEquals(student.getEmail(), studentData.getEmail());
+        assertEquals(student.getCourse(), studentData.getCourseId());
+        assertEquals(student.getName(), studentData.getName());
+
+        if (student.getLastName() == null) {
```

Comment:
Will `lastName` be null? I thought the DB will hide the complexity.

Reply:
DB does hide it. However, when I use `typicalBundle` and pull the student from the course, the `lastName` stays null. I wasnt sure how to get the proper version so I just did the logic to pull to `lastName` in the function itself.